### ISSUE TITLE
Feature Request: import support for aws_kinesis_firehose_delivery_stream

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#481]
_This issue was originally opened by @grep-stripe as hashicorp/terraform#11366. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

### Terraform Version
terraform -v
Terraform v0.8.4

### Affected Resource(s)
aws_kinesis_firehose_delivery_stream

### Terraform Configuration Files
The relevant portion of the configuration file.

```hcl
resource "aws_kinesis_firehose_delivery_stream" "flow_logs_delivery_stream" {
  name        = "flowlogs-kinesis-firehose-s3-stream"
  destination = "s3"

  s3_configuration {
    role_arn   = "${aws_iam_role.firehose_role.arn}"
    bucket_arn = "${aws_s3_bucket.flow_logs_s3_bucket.arn}"
  }
}
```

### Feature Request

Please support terraform import for the `aws_kinesis_delivery_stream` resource.

```
$ sc-terraform import aws_kinesis_firehose_delivery_stream.flow_logs_delivery_stream flowlogs-kinesis-firehose-s3-stream
aws_kinesis_firehose_delivery_stream.flow_logs_delivery_stream: Importing from ID "flowlogs-kinesis-firehose-s3-stream"...
Error importing: 1 error(s) occurred:

* import aws_kinesis_firehose_delivery_stream.flow_logs_delivery_stream (id: flowlogs-kinesis-firehose-s3-stream): resource aws_kinesis_firehose_delivery_stream doesn't support import
```

**Comment by docmerlin** [comment]
bump
If state gets messed up, it is annoying to fix without import

**Comment by bflad** [comment]
I believe this was just resolved in the next provider release by #2082.

**Comment by bflad** [comment]
This has been released in [terraform-provider-aws version 1.7.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#170-january-12-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Comment by bflad** [comment]
For anything relating to enhancing the other portions of importing this resource, please open new specific issues. Thanks!

**Comment by bflad** [comment]
v1.7.0 added the initial functionality and v1.7.1 (hopefully releasing tomorrow) will contain important bug fixes for this implementation (#2970, #3052). I'm going to close this issue out now, but please do not hesitate to open new issues with any additional bug reports or other enhancements.
